src/pages/about.astro

```diff
@@ -96,6 +111,44 @@ import FooterFull from '../components/footer/FooterFull.svelte'
           />
         </div>
       </div>
+      <div class="relative grid grid-cols-1 md:grid-cols-4 w-fit m-auto z-20">
+        {
+          metrics?.map((item) =>
+            item.data.metrics?.map((p) => (
+              <div class="text-center m-12">
+                <MetricCard icon={p.icon} value={p.value} metric={p.metric} />
+              </div>
+            ))
+          )
+        }
+      </div>
+      <!-- last section of the page, can't use ContentImageFull because of the CTA icon... -->
+      <div
+        class="items-center max-w-screen-xl gap-16 px-4 py-8 mx-auto lg:grid lg:grid-cols-2 md:py-8 md:px-6"
+      >
+        <div class="flex justify-center">
+          <img
+            class="w-3/4 rounded-lg aspect-square object-cover md:h-[550px] md:w-[550px] md:min-w-[550px]"
+            src="images/mcswf_marine_sgt.png"
+          />
+        </div>
+        <div class="font-light text-white sm:text-lg">
+          <div class="flex justify-center">
```

Comment:
Should probably add padding.

Reply:
I thought the spacing looked good, not sure how it'll look once the metric cards have borders but maybe we can merge this and then make that change? or if you think the spacing is off just let me know how much and I can adjust